torchgen/model.py

```diff
@@ -1183,6 +1187,20 @@ def strip_ret_annotation(r: Return) -> Return:
         if strip_view_copy_name and base_name.endswith("_copy"):
             base_name = base_name.replace("_copy", "")
 
+        if convert_mutable_inputs_to_returns:
+            # find mutable inputs that are not originally returned, and conver them to returns
+            returns_from_mutable_inputs = tuple(
+                Return(name=None, type=a.type, annotation=None)
+                for a in self.arguments.flat_all
+                if a.annotation is not None
+                and a.annotation.is_write
+                and not any(a.annotation == r.annotation for r in self.returns)
+            )
+            original_returns = tuple(map(strip_ret_annotation, self.returns))
+            returns = returns_from_mutable_inputs + original_returns
```

Comment:
you want the returns from mutable inputs to come first? That's actually the opposite of what I expected

Reply:
oh haha I wasn't sure which one would be clearer, so I just picked them going first. But yeah I can kinda see it being clearer the other direction - I'll switch to that.